packages/serenity-bdd/package.json

```diff
@@ -38,7 +38,8 @@
     "clean": "rimraf target",
     "test": "c8 npm run test:no-coverage",
     "test:no-coverage": "mocha --config ../../.mocharc.yml 'spec/**/*.spec.*'",
-    "compile": "rimraf lib && tsc --project tsconfig.build.json"
+    "compile": "rimraf lib cache && tsc --project tsconfig.build.json && npm run populate-cache",
```

Comment:
> I'd suggest to use rimraf lib instead of rimraf lib cache and keep the local cache since the jars are immutable and don't need to be redownloaded

Good point, makes sense to me, will adjust.

> Populating the cache feels like a separate step to compile, I'd suggest removing the explicit call to `npm run populate-cache`

I'm mot entirely sure I get the idea, please correct me if I'm wrong. You propose to explicitly add `populate-cache` step in the build process in the [Makefile](https://github.com/serenity-js/serenity-js/blob/cc6e399a6ec5433e7470eb8bd0267479f50c4989/Makefile#L40) and in the [CI](https://github.com/serenity-js/serenity-js/blob/cc6e399a6ec5433e7470eb8bd0267479f50c4989/.github/workflows/main.yaml#L29)?